README.md

```diff
@@ -1,7 +1,28 @@
-# FortiGate Autoscale Core module
-This is the Autoscale Core module written in TypeScript.
+# Autoscale Core
+The Core Library for the Fortinet Autoscale project.
 
-Note: This is a pre-release for internal development purposes.
+The Autoscale project provides a cloud-based multi-group auto scaling functionality for virtual machines of Fortinet products which have native HA features to form a cluster with failover protection.
+
+This project provides the fundamental, generic logics of a Hybrid Licensing architecture. It also includes implementations for certain popular cloud platforms.
```

Comment:
The noun *logic* can be countable or uncountable. In more general, commonly used, contexts, the plural form will also be *logic*.

Reply:
Kimberly's change has resolved it.